docs/D-Modularity/functions-arrays-and-structs.md

```diff
@@ -813,37 +831,40 @@ The table includes:
 * the type of each member
 * the name of each member
 
->Note the breakdown of each object into its members in the head of the table.  We reserve a separate line for the addresses that are pointed to:
-
+:::note
+Note the breakdown of each object into its members in the head of the table.  We reserve a separate line for the addresses that are pointed to:
```

Comment:
Get rid of the word "Note" since the whole thing is now a note.  Should be, "The breakdown of each..."

Reply:
Done